server/models/chatrooms.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const ChatRoomSchema = new Schema({
+  userChatId: {
+    type: String,
+    required: true,
+  },
+  userChatId2: {
+    type: String,
+    required: true,
+  },
+  roomMessages: {
+    type: Array,
+    default: [],
+  },
+  roomUser1: {
```

Comment:
are `roomUser1` and `roomUser2` used?

Reply:
Yes roomUser1 and 2 are needed. Chat room is exclusive to only 2 users, hence I needed id and name of both. Let me know if there is a better approach. 
That being said, userChatId1 and 2 are obsolete. I have already removed those fields from code base. Just need to remove it from model.